docs/source/getting-started/quickstart-2/index.rst

```diff
@@ -197,65 +220,99 @@ Choose **Chart view**. Choose the **Parallel coordinates** graph and configure i
     alt="Screenshot of MLflow tracking UI parallel coordinates graph showing runs"
   />
 
-The red graphs on this graph are runs that fared poorly. The lowest one is a baseline run with both **lr** and **momentum** set to 0.0. That baseline run has an RMSE of ~0.89. The other red lines show that high **momentum** can also lead to poor results with this problem and architecture. 
+The red graphs on this graph are runs that fared poorly. The lowest one is a baseline run with both **lr** 
+and **momentum** set to 0.0. That baseline run has an RMSE of ~0.89. The other red lines show that 
+high **momentum** can also lead to poor results with this problem and architecture. 
 
 The graphs shading towards blue are runs that fared better. Hover your mouse over individual runs to see their details.
 
 Register your best model
 ------------------------
 
-Choose the best run and register it as a model. In the **Table view**, choose the best run. In the **Run Detail** page, open the **Artifacts** section and select the **Register Model** button. In the **Register Model** dialog, enter a name for the model, such as ``wine-quality``, and click **Register**.
+Choose the best run and register it as a model. In the **Table view**, choose the best run. In the 
+**Run Detail** page, open the **Artifacts** section and select the **Register Model** button. In the
+**Register Model** dialog, enter a name for the model, such as ``wine-quality``, and click **Register**.
+
+Now, your model is available for deployment. You can see it in the **Models** page of the MLflow UI.
+Open the page for the model you just registered.
 
-Now, your model is available for deployment. You can see it in the **Models** page of the MLflow UI. Open the page for the model you just registered.
+You can add a description for the model, add tags, and easily navigate back to the source run that generated
+this model. You can also transition the model to different stages. For example, you can transition the model
+to **Staging** to indicate that it is ready for testing. You can transition it to **Production** to indicate
+that it is ready for deployment.
 
-You can add a description for the model, add tags, and easily navigate back to the source run that generated this model. You can also apply aliases and tags to the model for easier organization and deployment. For more information, see :ref:`registry`.
+Transition the model to **Staging** by choosing the **Stage** dropdown:
+
+.. image:: ../../_static/images/quickstart_mlops/register_model_button.png
+    :width: 800px
```

Comment:
Can we set this to a percentage value? 70% at full screen will render this to roughly what 800px would be. Fixed width sizes have issues with screen resizing.